### ISSUE TITLE
[INSTALLATION HELP]

### ISSUE THREAD

**Issue by ckluong** [kushalkolar/MESmerize#104]
Please try these things before posting an installation issue:

1. Create a fresh new anaconda or virtual environment.

2. If you're having difficulties with `caiman` installation try installing a specific version, for example:
  `conda install -c conda-forge caiman==1.8.5`

3. If you want to use caiman features, `caiman` _must_ but installed in the same environment as `mesmerize`.

4. If you want to use certain features like kshape clustering or cross-correlation, `tslearn` must be installed in the same environment as `mesmerize`.

Otherwise use this template:

**System information**
1. OS (MacOS Monterey 12.4)
2. CPU (2.6 GHz 6-Core Intel Core i7)
3. RAM 16GB 2667 MHz DDR4
4. SSD

**Copy & paste the entire output of the terminal while you were trying to install when the issue came up:**
I have tried conda install -c conda-forge caiman==1.8.5 as well and it infinitely solved and never ended. I tried the mamba method and got this output.
```
(mesmerize) christopherluong@Christophers-Mac CaImAn % mamba env create -f environment.yml -n caiman

conda-forge/osx-64       Using cache
conda-forge/noarch       Using cache
pkgs/main/osx-64         [====================] (00m:00s) No change
pkgs/main/noarch         [====================] (00m:00s) No change
pkgs/r/osx-64            [====================] (00m:00s) No change
pkgs/r/noarch            [====================] (00m:00s) No change


Looking for: ["python[version='>=3.10']", 'bokeh', 'coverage', 'cython', 'future', 'h5py', 'holoviews', 'ipykernel', 'ipython', 'ipyparallel', 'jupyter', 'matplotlib', 'mypy', 'nose', 'numpy', 'numpydoc', 'opencv', 'peakutils', 'pims', 'psutil', 'pynwb', 'pyqtgraph', "scikit-image[version='>=0.19.0']", 'scikit-learn', 'scipy', "tensorflow[version='>=2.4.0']", 'tifffile', 'tk', 'tqdm', 'yapf', 'z5py']


Encountered problems while solving.
Problem: nothing provides __osx >=10.10 needed by grpcio-1.39.0-py37hac9a2c0_0```

**Installation method**
1. Are you using anaconda, python virtual environments, or something else?
  -> conda version 4.12.0

2. If you're installing mesmerize through pip, what version of pip is installed in the environment?
-> pip version 21.0.1

3. Are you trying to install in developer mode from the repo? No

4. What is the python version in the environment you're trying to install `mesmerize` in? 
-> python version 3.8.8

5. Information if you're using some other installation method.

**Comment by ckluong** [comment]
I think solved by using mamba install -c conda-forge caiman.

I then got an error running mesmerize, but it was solved by downgrading pandas to 1.1.0

I now get:

(mesmerize) christopherluong@Christophers-Mac ~ % mesmerize
/opt/anaconda3/envs/mesmerize/lib/python3.6/site-packages/sklearn/utils/deprecation.py:143: FutureWarning: The sklearn.cluster.k_means_ module is  deprecated in version 0.22 and will be removed in version 0.24. The corresponding classes / functions should instead be imported from sklearn.cluster. Anything that cannot be imported from sklearn.cluster is now part of the private API.
  warnings.warn(message, FutureWarning)

Is this ok? Also how sensitive is this to updating python to say 3.9?
